orders/views.py

```diff
@@ -1,3 +1,65 @@
-from django.shortcuts import render
+import json
+
+from django.http.response  import JsonResponse
+from django.views          import View
+
+from .models               import Cart
+
+class CartView(View):
+    def get(self, request):
+            carts  = Cart.objects.filter(user_id = request.user_id)
+            cart_items = []
+            for cart in carts:
+                cart_items = [{
+                    "product_id"      : cart.product.id,
+                    "product_kr_name" : cart.product.kr_name,
+                    "product_en_name" : cart.product.en_name,
+                    "product_price"   : cart.product.price,
+                    "product_image"   : [image.url for image in cart.product.image_set.all()]
+                }]
+            return JsonResponse({'cart_items' : cart_items}, status=200)
+
+    def post(self, request):
+        try:
+            data               = json.loads(request.body)
+            Cart.objects.create(
+                product_id     = data['product_id'],
+                user_id        = request.user.id,
```

Comment:
suggestion과 같이 바로 user instance를 입력해주셔도 됩니다.
```suggestion
                user           = request.user,
```